framework/db/mssql/QueryBuilder.php

```diff
@@ -661,5 +666,4 @@ public function dropColumn($table, $column)
         return $this->dropConstraintsForColumn($table, $column) . "\nALTER TABLE " . $this->db->quoteTableName($table)
             . " DROP COLUMN " . $this->db->quoteColumnName($column);
```

Comment:
```suggestion
        return $this->dropConstraintsForColumn($table, $column) . "\nALTER TABLE " . $tableName
            . " DROP COLUMN " . $columnName;
```

Reply:
It should be quoted

Reply:
Yes, aren't the variables already having it?

Reply:
I'm not correct function `dropColumn`
```php
    /**
     * Drop all constraints before column delete
     * {@inheritdoc}
     */
    public function dropColumn($table, $column)
    {
        return $this->dropConstraintsForColumn($table, $column) . "\nALTER TABLE " . $this->db->quoteTableName($table)
            . " DROP COLUMN " . $this->db->quoteColumnName($column);
    }
```

Reply:
![2021-04-28_19-21-04](https://user-images.githubusercontent.com/8342004/116438421-1867a980-a857-11eb-97df-4cb620eb494d.png)

Reply:
In function alterColumn - fixed
```php
        return 'ALTER TABLE ' . $tableName . ' ALTER COLUMN '
            . $columnName . ' '
            . $this->getColumnType($type) . "\n"
            . implode("\n", $sqlAfter);
```

Reply:
Ohhh, I messed up the code in two methods 😀 sorry